README.md

```diff
@@ -21,91 +21,72 @@ take effect the handstroke after they are called, and `Bob` and `Single` will re
 - Automatic correction for Ringing Room using multiple server URLs for load balancing
 - Very light resource footprint, as it doesn't interact with a browser to talk to Ringing Room
 
-### Roadmapped features
-- Make the bot able to ring touches of methods
+### Contributing
+Contributions are very welcome!  See [here](CONTRIBUTING.md) for guidance.
 
 ## Quickstart
-### Installation (same for all platforms if using the command line)
-1. Clone and download this repository (using
-   [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) will make updating easier):
+### Installation (using Pip)
+1. Run the following command to install or update to the latest version of the bot:
    ```bash
-   git clone https://github.com/Kneasle/ringing-room-bot
+   pip3 install --upgrade rr-bot
```

Comment:
To confirm `--upgrade` is valid on the initial install?

Reply:
Yep it is.